### DISCUSSION TITLE
How to formatISO without time zone ?

### DISCUSSION THREAD

**Discussion by aspirisen** [js-fns/date-fns#2834] [Q&A]
Hello, I haven't found such in docs, how to format date to ISO without Time Zone information

Current behavior:
```
formatISO(new Date(2019, 8, 18, 19, 0, 52), { representation: 'time' })
// '19:00:52+03:00'
```

Expected behavior:
```
formatISO(new Date(2019, 8, 18, 19, 0, 52), { representation: 'time', timeZone: false })
// '19:00:52'
```

**Comment by fturmel** [comment]
@aspirisen There are no combination of options for `formatISO` right now that will give you only time without a timezone.

I suggest you try [lightFormat](https://date-fns.org/docs/lightFormat) instead:

```js
import { lightFormat } from "date-fns";

const res = lightFormat(new Date(2019, 8, 18, 19, 0, 52), "HH:mm:ss");
console.log(res); // 19:00:52
```

**Reply by aspirisen** [reply]
Yeah, but the difficulty is that i need to specify possible millisecond for template, also I have a case where i need date+time without timezone etc., the possible combination of patterns is pretty big. 

But anyway, thanks for the answer
